index.js

```diff
@@ -146,10 +146,27 @@ const plugin = fp(async function (app, opts) {
     schema = gateway.schema
     entityResolversFactory = gateway.entityResolversFactory
 
+    let gatewayInterval = null
+
+    if (gateway.pollingInterval !== undefined) {
```

Comment:
Shouldn't this better test for a numeric value like `typeof gateway.pollingInterval === 'number'` or similar?
I was just thinking that it's an easy mistake to apply `null` or `false` to the config option & given how `setInterval` treats this values (falling back to it's default of `0` for the delay afaik) it would go ahead & call the function a gazillion without any delay executing tons of requests, bringing the attached services to their knees in the worst case.

Reply:
Yea, I agree. I will change it, thanks :+1: